packages/cardpay-sdk/sdk/native-currencies.ts

```diff
@@ -11,6 +11,50 @@ export default {
     smallThreshold: 0.003,
     symbol: 'Ξ',
   },
+  DAI: {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'DAI',
+    decimals: 18,
+    label: 'Dai',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
+    symbol: '',
+  },
+  CARD: {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'CARD',
+    decimals: 18,
+    label: 'CARD',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
+    symbol: '',
+  },
+  'DAI.CPXD': {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'DAI.CPXD',
+    decimals: 18,
+    label: 'Dai.CPXD',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
+    symbol: '',
+  },
+  'CARD.CPXD': {
+    alignment: 'left',
+    assetLimit: 0.001,
+    currency: 'CARD.CPXD',
+    decimals: 18,
+    label: 'CARD.CPXD',
+    mask: '[09999999999]{.}[999999999999999999]',
+    placeholder: '0',
+    smallThreshold: 0.003,
+    symbol: '',
+  },
```

Comment:
Let's remove the CPXD tokens. The CPXD tokens are really just analogs for their mainnet counterparts. There will only ever be one way to format a CARD or DAI token regardless of whether it is the layer1 version or the layer 2, CPXD version.

Reply:
ya I originally didn’t have them but I wanted the validation string to match the symbol displayed in the field and I hesitated to use hackish-seeming string replacement:

![image](https://user-images.githubusercontent.com/43280/133641730-1d1237de-60a1-4a54-a12d-1ede83168dda.png)
 
I could add an optional `currencyDisplayOverride` parameter, or do you think I should just change the string after I get it back?

Reply:
Hmmm, I guess I don’t have a real strong feeling either way. And I assume the UI designs are actually requesting that we show the .CPXD suffix in the text fields?

Reply:
ya, exactly. I think I’ll just use string replacement for now and see what people think 😬 thanks